core/src/main/resources/error/error-classes.json

```diff
@@ -520,6 +520,11 @@
           "NATURAL CROSS JOIN."
         ]
       },
+      "OPERATION_ONLY_SUPPORTED_WITH_V2_TABLE" : {
```

Comment:
let's make it short as it's a sub-error-class. How about `TABLE_OPERATION`

Reply:
Sounds good. Changed.